src/app/store.js

```diff
@@ -1,10 +1,11 @@
 import { configureStore } from '@reduxjs/toolkit';
 import authReducer from '../features/authSlice';
+import pollReducer from "../features/pollSlice";
```

Comment:
create a folder name reducer and move all reduces into that, not in a features folder @kaushaki-kumari